packages/components/src/Tree/Tree.tsx

```diff
@@ -83,39 +87,51 @@ const indicatorProps: AccordionIndicatorProps = {
   indicatorSize: 'small',
 }
 
+export const InnerTreeItem = styled(TreeItem)``
```

Comment:
To keep everything "named-spaced" within `Tree` can we flip the naming a bit here?

```suggestion
export const TreeItemInner = styled(TreeItem)``
```

Reply:
Resolved in 48f100f!